### PR TITLE
move further reading, convert to rst, remove dead links

### PR DESCRIPTION
**Pull request by aikaterine**
Moved further reading file into docs, converted from md to rst, removed dead links

### CODE REVIEW

**Review by TimoBakx — APPROVED** [review]

File: contributing/diversity/further_reading.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Further reading / viewing
+====================
```
Comment (OskarStark) [inline contributing/diversity/further_reading.rst:2]: Please match the length of the underline, the length of the headline.

Same for the others, thanks

**Comment by OskarStark** [comment]
Thank you, I merged it in 5.4 branch and fixed the underline length. Cheers

**Comment by aikaterine** [comment]
Thank you! I haven't contributed to Symfony before so still getting  head around the reqs :)

**Comment by OskarStark** [comment]
All good, thanks 👍

**Pull request merged**